boards/ek-lm4f120xl/Makefile.include

```diff
@@ -0,0 +1,23 @@
+# define the cpu used by the ek-lm4f120xl board
+export CPU = lm4f120
+export CPU_MODEL = lm4f120
```

Comment:
This should be `lm4f120h5`, h -> 256k flash, 5 -> 32K RAM. These infos are needed for selecting the correct linkerscript...